src/server/api/schema.js

```diff
@@ -556,6 +557,38 @@ const rootMutations = {
       return updatedUser;
     },
 
+    setUserSuspended: async (_root, { userId, isSuspended }, { user, db }) => {
+      await superAdminRequired(user);
+
+      await db.primary.transaction(async (trx) => {
+        await trx.raw(`update public.user set is_suspended = ? where id = ?`, [
+          isSuspended,
+          userId
+        ]);
+
+        await trx.raw(`delete from user_session where user_id = ?`, [userId]);
```

Comment:
Is it ok to do this even if what we're doing here is "unsuspending"? I think so, but just thinking aloud